participant/tests/test_create_new_req_views.py

```diff
@@ -0,0 +1,179 @@
+from rest_framework.reverse import reverse
+from django.test import Client, TestCase
+from rest_framework import status
+import json
+from datahub.models import Organization, UserOrganizationMap
+from accounts.models import User, UserRole
+from participant.models import SupportTicketV2
+from rest_framework_simplejwt.tokens import RefreshToken
+from django.core.files.uploadedfile import SimpleUploadedFile
+from test_util import TestUtils
+
+auth_co_steward= {
+    "token": "null"
+}
+
+auth_participant={
+   "token": "null" 
+}
+
+
+
+class  SupportTicketNewRequestTestCaseForViews(TestCase):
+
+    def setUp(self) -> None:
+        self.client = Client()
+        self.client_participant = Client()
+        self.support_ticket_url = reverse("support_tickets-list")
+        self.user_role_participant = UserRole.objects.create(id=3, role_name="datahub_participant_root")
+        self.user_costeward_role = UserRole.objects.create(id=6, role_name="datahub_co_steward")
+        self.user_participant = User.objects.create(
+            email="akshata@digitalgreen.org",
+            role_id=self.user_role_participant.id,
+        )
+        self.user_co_steward = User.objects.create(
+            email="costeward@digitalgreen.org",
+            role_id=self.user_costeward_role.id,
+        )
+        self.creating_org_participant = Organization.objects.create(
+            org_email="participant@digitalgreen.org",
+            name="Participant One",
+            phone_number="5678909876",
+            website="htttps://google.com",
+            address=({"city": "Banglore"}),
+        )
+        self.creating_co_steward = Organization.objects.create(
+            org_email="costewdardorg@digitalgreen.org",
+            name="Co-steward",
+            phone_number="1234567890",
+            website="htttps://google.com",
+            address=({"city": "Banglore"}),
+        )
+        self.user_co_steward_map=UserOrganizationMap.objects.create(user_id=self.user_co_steward.id, organization_id=self.creating_co_steward.id)
+        self.user_map_participant=UserOrganizationMap.objects.create(user_id=self.user_participant.id, organization_id=self.creating_org_participant.id)
+
+
+        # Access token for Co-steward
+        auth_co_steward["token"] =  TestUtils.create_token_for_user(self.user_co_steward, self.user_co_steward_map)
+        co_steward_header=self.set_auth_headers(co_steward=True)
+        self.client.defaults['HTTP_AUTHORIZATION'] = co_steward_header[0]
+        self.client.defaults['CONTENT_TYPE'] = co_steward_header[1]
+
+
```

Comment:
Please add the test cases for co-steward where he can see only tickets raised by his participant.